compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -763,6 +763,7 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     // ==========================================================================
 
     rustc_attr!(TEST, rustc_access_level, Normal, template!(Word), WarnFollowing),
+    rustc_attr!(TEST, rustc_effective_visibility, Normal, template!(Word), WarnFollowing),
```

Comment:
No need for two testing attributes, one should be enough, it can print what `#[rustc_effective_visibility]` prints now, i.e. the extended information.